vulpes/hooks/save_load.cpp

```diff
@@ -73,11 +76,13 @@ void init_save_load_hook() {
         auto function_pointers = reinterpret_cast<fp_type*>(*sig_hook_save_load());
         // All of these are stored in arrays that in both 1.10 client and
         // server (our only targets) are right next to each other.
-        before_save_proc_orig = function_pointers[0];
-        before_load_proc_orig = function_pointers[1];
-        after_load_proc_orig  = function_pointers[14];
+        before_save_proc_orig = function_pointers[0];  // The only before_save function pointer.
+        before_load_proc_orig = function_pointers[1];  // The only before_load function pointer.
+        after_load_proc_orig  = function_pointers[14]; // The last after_load function pointer.
```

Comment:
I still think 0, 1, and 14 should be named constants. If these function pointers are all constant, well-known things, you could even break the pointers themselves out into an enum or a struct or something a bit more abstract than an array.